pkg/apis/v1alpha1/register.go

```diff
@@ -53,11 +53,12 @@ var (
 		AMIFamilyAL2:          sets.NewString("dockerd", "containerd"),
 		AMIFamilyUbuntu:       sets.NewString("dockerd", "containerd"),
 	}
-	ResourceNVIDIAGPU   v1.ResourceName = "nvidia.com/gpu"
-	ResourceAMDGPU      v1.ResourceName = "amd.com/gpu"
-	ResourceAWSNeuron   v1.ResourceName = "aws.amazon.com/neuron"
-	ResourceHabanaGaudi v1.ResourceName = "habana.ai/gaudi"
-	ResourceAWSPodENI   v1.ResourceName = "vpc.amazonaws.com/pod-eni"
+	ResourceNVIDIAGPU        v1.ResourceName = "nvidia.com/gpu"
+	ResourceAMDGPU           v1.ResourceName = "amd.com/gpu"
+	ResourceAWSNeuron        v1.ResourceName = "aws.amazon.com/neuron"
+	ResourceAWSNitroEnclaves v1.ResourceName = "aws.ec2.nitro/nitro_enclaves"
```

Comment:
`aws.ec2.nitro` is odd to me, since typically these are resolvable FQDNs. Seems like we're stuck with what was chosen upstream. `nitro_enclaves` should be kabob-case too